backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/VersionController.kt

```diff
@@ -37,7 +37,7 @@ class VersionController constructor(private val versionHandler: VersionHandler,
             logger.info("Specification $specificationId version $version requested as YAML")
             val jsonNodeTree = ObjectMapper().readTree(foundVersion.content)
             val jsonAsYaml = YAMLMapper().writeValueAsString(jsonNodeTree)
-            return versionDtoMapper.fromDomain(Version(foundVersion.version, jsonAsYaml))
+            return versionDtoMapper.fromDomain(Version(foundVersion.version, jsonAsYaml, foundVersion.language))
```

Comment:
Does it make sense to render GraphQL as yaml? Is it not something completely different from json and yaml?

Reply:
GraphQL isn't YAML, and this `if` block is not reached for GraphQL. But the `Version` constructor now requires a language parameter. Maybe I haven't understood your question?